apps/elewa-group-website/src/_variables.scss

```diff
@@ -45,4 +45,6 @@
   --elewa-group-website-font-size-description: 13px;
   --elewa-group-website-font-size-description-medium: 12px;
 
+  //reusable team member view
+
 }
```

Comment:
This file did not need to change

Reply:
this has been resolved using git rebase -i <hash>

Reply:
Sending new PR to solve this